src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        excavator.doWork();
+        excavator.stopWork();
+
+        Bulldozer bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        bulldozer.stopWork();
+
+        Truck truck = new Truck();
+        truck.doWork();
+        truck.stopWork();
```

Comment:
this code violates task requirements
![image](https://user-images.githubusercontent.com/11480043/227714832-ec8b2d97-fabf-46f7-8853-a7a763da6bc8.png)